src/gan/gen_pyt/asdl/asdl_ast.py

```diff
@@ -0,0 +1,213 @@
+# coding=utf-8
+
+from io import StringIO
+
+from multivac.src.gan.gen_pyt.asdl import *
```

Comment:
let's ensure -- for any file -- that repo-based imports like this will work as-is with docker.